rotkehlchen/chain/evm/decoding/hop/decoder.py

```diff
@@ -333,13 +342,77 @@ def _decode_add_liquidity(self, context: DecoderContext) -> DecodingOutput:
             elif (
                 event.event_type == HistoryEventType.RECEIVE and
                 event.event_subtype == HistoryEventSubType.NONE and
-                event.location_label == user_address
+                event.location_label == user_address and
+                event.asset.is_evm_token()
             ):
                 event.event_type = HistoryEventType.RECEIVE
                 event.event_subtype = HistoryEventSubType.RECEIVE_WRAPPED
                 event.counterparty = CPT_HOP
                 event.notes = f'Receive {event.balance.amount} {event.asset.symbol_or_name()} after providing liquidity in Hop'  # noqa: E501
                 in_event = event
+                lp_token = event.asset.resolve_to_evm_token()
+                GlobalDBHandler.edit_evm_token(
+                    entry=EvmToken.initialize(
+                        address=lp_token.evm_address,
+                        chain_id=self.evm_inquirer.chain_id,
```

Comment:
All this new logic can you move it to a method for populating the pool in the database? I believe it would benefict from having a proper docstring and a good function name with the errors that it can raise